code/obj/item/storage/ammo_pouches.dm

```diff
@@ -192,7 +192,7 @@
 	spawn_contents = list(/obj/item/handcuffs = 3,\
 	/obj/item/device/flash,\
 	/obj/item/reagent_containers/food/snacks/donut,\
-	/obj/item/instrument/whistle)
+	/obj/item/instrument/whistle/security)
```

Comment:
Should this be in the brobocop module, too?

Reply:
I'm putting it in before someone tells me not to.